earth_enterprise/src/fusion/rasterfuse/TmeshGenerator.cpp

```diff
@@ -74,14 +77,17 @@ TmeshGenerator::Generate(float *srcSamples,
   elev.resize(SizeX * SizeY);
 
   // Error thresholds for decimation - set these to 0 to skip decimation
-  // jdjohnso - These numbers came from empirical testing?
-  // jdjohnso - The allowable error gets smaller as the resolution increases.
-  // jdjohnso - The allowable error on border tiles is 1/10 that of interior
+  // Question - Did these numbers come from empirical testing?
+  // The allowable error gets smaller as the resolution increases.
+  // The allowable error on border tiles is 1/10 that of interior
   // tiles. This is because we really want the border tiles to match. If not
   // we get rips between mesh tiles in the client.
-  int error_level = tmeshAddr.level+1;
-  double error = 0.009 * pow(2, 11-error_level);
-  double borderError = error / 10.0;
+  //int error_level = tmeshAddr.level+1;
```

Comment:
We should remove this line and the next.

Reply:
Good point..  Just removed them